--- ginf/src/Makefile.am.old	Sun Jul 14 21:32:27 2002
+++ ginf/src/Makefile.am	Mon Sep 30 18:34:26 2002
@@ -2,7 +2,8 @@
 
 INCLUDES = \
 	-I$(top_srcdir)/intl \
-	$(GNOME_INCLUDEDIR)
+	$(GNOME_INCLUDEDIR) \
+	`gdk-pixbuf-config --cflags`
 
 bin_PROGRAMS = ginf
 
@@ -13,5 +14,6 @@
 	callbacks.c callbacks.h \
 	functions.c functions.h 
 
-ginf_LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS)
+ginf_LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS) -lgtkhtml
+ginf_LDFLAGS = `gdk-pixbuf-config --libs`
 
